src/config.rs

```diff
@@ -304,6 +316,33 @@ impl Context {
                 let value = value.map(improve_single_line_input);
                 self.sql.set_raw_config(key, value.as_deref()).await?;
             }
+            Config::DebugLogging => {
+                if value == Some("0") || value == Some("") || value == None {
+                    if let Some(webxdc_message_id) =
+                        self.sql.get_raw_config_u32(Config::DebugLogging).await?
+                    {
+                        message::delete_msgs(self, &[MsgId::new(webxdc_message_id)]).await?;
+                    }
+                    self.sql.set_raw_config(key, value).await?;
+                    self.debug_logging.store(0, atomic::Ordering::Relaxed);
+                } else {
+                    // the unbundled version lives at https://github.com/webxdc/webxdc_logging
```

Comment:
What happens if set_config("debug_logging", 1) is called twice?

Reply:
What if the code calls this?
```
dc_set_config(context, "debug_logging", 1);
dc_set_config(context, "debug_logging", 1);
```

Maybe Android UI doesn't do this, but some UI (Desktop maybe?) may save all the settings each time you hit "Save" in the dialog. The code looks like it will insert a new message each time even if there is already a previous one.

Reply:
> some UI may save all the settings each time you hit "Save" in the dialog.

Ah yeah, that would be a problem; I changed this.